docs/source/networks.rst

```diff
@@ -35,6 +35,21 @@ Blocks
 .. autoclass:: SEBlock
     :members:
 
+`Squeeze-and-Excitation Bottleneck`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. autoclass:: SEBottleneck
+    :members:
+
+`Squeeze-and-Excitation Resnet Bottleneck`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. autoclass:: SEResNetBottleneck
+    :members:
+
+`Squeeze-and-Excitation Resnext Bottleneck`
```

Comment:
I feel it should be `Squeeze-and-Excitation ResneXt Bottleneck`

Reply:
Thanks, fixed.